src/localauthentication.cs

```diff
@@ -56,19 +56,19 @@ interface LAContext {
 		[NullAllowed, Export ("localizedCancelTitle")]
 		string LocalizedCancelTitle { get; set; }
 
-#if !MONOMAC
-		[Availability (Introduced = Platform.iOS_8_3, Deprecated = Platform.iOS_9_0)]
-		[NullAllowed]
-		[Export ("maxBiometryFailures")]
-		NSNumber MaxBiometryFailures { get; set; }
-
-		[iOS (9,0)]
+		[iOS (9,0)][Mac (10,12,3)]
```

Comment:
Also the type is decorated with `onlyOn64 : true` which should be added to any member attribute (others are missing too).